addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -235,26 +235,28 @@ PortalChatter.include({
     // Handlers
     //--------------------------------------------------------------------------
 
+    _onChangeDomain: function () {
+        // Show spinner and hide messages during loading
```

Comment:
I would add that it's an override (and maybe btw put the comment as description in the JSDoc?).

Reply:
Good idea, added